packages/react-saasify/src/components/LiveServiceDemo/LiveServiceDemo.js

```diff
@@ -105,60 +127,69 @@ export class LiveServiceDemo extends Component {
               />
             </div>
           ))}
+        </div>
 
-          {output && (
-            <Fragment>
-              <Divider />
+        <div className={theme(styles, 'footer')}>
 
-              <div
-                className={theme(styles, 'output')}
-              >
-                {renderedOutput}
+          <div className={theme(styles, 'footer__service')}>
+            {this._example.description && (
+              <div className={theme(styles, 'footer__service__example-description')}>
```

Comment:
I'm not a fan of having singleton descendent selectors because it can make debugging difficult. I know CSS modules makes it possible, but when I'm inspecting in the browser and I see `module_content_xy23` and `module_content_4tg2` (pseudocode), I can't tell where they're getting their styles from. Seeing `module_section_content_xy23` and `module_nav-footer_content_4tg2`, I can now figure it out.

Having said that, I don't really like the above BEM style of CSS naming, it was just closer to what we already have. My preference for React components is [SUIT CSS](https://suitcss.github.io) or `styled-components`.

Reply:
Ok sounds good -- let's chat about options sometime soon.

Reply:
Ahh also, we should tweak the CSS module settings to make things more readable.